examples/checkout/index.js

```diff
@@ -0,0 +1,70 @@
+$(function() {
+
+  /* Build new ShopifyBuy client
+  ============================================================ */
+  var client = ShopifyBuy.buildClient({
+    apiKey: 'bf081e860bc9dc1ce0654fdfbc20892d',
+    myShopifyDomain: 'embeds',
+    appId: '6'
+  });
+
+  var product;
+  var cart;
+
+  client.createCart().then(function (newCart) {
+    cart = newCart;
+    return client.fetchProduct('3614436099');
```

Comment:
Does this need to be chained after `createCart` like this? Could they be in parallel? Right now the cart is basically instantaneous, but in the future these will be separate requests. 
```javascript
var cartPromise = client.createCart();
var productPromise = client.fetchProduct('3614436099');
Promise.all([cartPromise, productPromise]).then(function(values) {
  cart = values[0];
  product = values[1];
  cart.addVariants(...
});
```